.gitignore

```diff
@@ -1,5 +1,8 @@
 build/build/
 build/temp/
 build/zap-exts/
+build/results/
```

Comment:
This entry is already added in the (upstream) beta branch (a rebase should take care of that).